squad-server/plugins/db-log.js

```diff
@@ -430,6 +475,11 @@ export default class DBLog extends BasePlugin {
     this.server.on('PLAYER_WOUNDED', this.onPlayerWounded);
     this.server.on('PLAYER_DIED', this.onPlayerDied);
     this.server.on('PLAYER_REVIVED', this.onPlayerRevived);
+    this.server.on('PLAYER_CONNECTED', this.onPlayerConnected);
+    this.server.on('PLAYER_DISCONNECTED', this.onPlayerDisconnected);
+
+    for (const steamID of Object.keys(this.playerSession))
+      await this.startSession(steamID, this.playerSession[steamID].name, new Date());
```

Comment:
I think I would scrap this and just track what happens whilst the plugin is mounted.

I can see issues arising because of it, e.g. if SquadJS crashes and reboots everyone is marked with a new session time,